python/guide/performance/compression/float16.md

```diff
@@ -0,0 +1,243 @@
+
+## Prerequisites
+
+- Volta range of Nvidia GPUs (e.g. AWS P3 instance)
+- Cuda 9 or higher
+- CUDNN v7 or higher
+
+This tutorial also assumes understanding of how to train a network with float32. Please refer to other tutorials [here](http://beta.mxnet.io/guide/index.html) to get started with Apache MXNet and Gluon API. This tutorial focuses on the changes needed to switch from float32 to mixed precision and tips on achieving the best performance with mixed precision.
+
+## Using the Gluon API
+
+### Training or Inference
+
+With Gluon API, you need to take care of three things to convert a model to support float16.
+
+1. Cast Gluon Block's parameters and expected input type to float16 by calling the [cast](http://beta.mxnet.io/api/gluon/mxnet.gluon.nn.Block.html#mxnet.gluon.nn.Block.cast) method of the Block representing the network.
+
+```
+net = net.cast('float16')
+```
+
+2. Ensure the data input to the network is of float16 type. If your DataLoader or Iterator produces output in another datatype, then you would have to cast your data. There are different ways you can do this. The easiest would be to use the [astype](http://beta.mxnet.io/api/ndarray/_autogen/mxnet.ndarray.NDArray.astype.html#mxnet.ndarray.NDArray.astype) method of NDArrays.
+
+```python
+data = data.astype('float16', copy=False)
+```
+
+If you are using images and DataLoader, you can also use a [Cast transform](http://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.data.vision.transforms.Cast.html).
+
+3. It is preferable to use **multi_precision mode of optimizer** when training in float16. This mode of optimizer maintains a master copy of weights in float32 even when the training (i.e. forward and backward pass) is in float16. This helps increase precision of the weight updates and can lead to faster convergence for some networks.
```

Comment:
of the weights

Reply:
can lead to faster convergence. (Is it for some networks or in some scenarios).

Reply:
I guess scenarios are more generic term, so I put scenarios.